procto.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
   gem.extra_rdoc_files = %w[LICENSE README.md CONTRIBUTING.md]
   gem.license          = 'MIT'
 
-  gem.add_development_dependency 'bundler', '~> 1.6.1'
+  gem.add_development_dependency 'bundler', '~> 1.11.2'
```

Comment:
Heh, now that you mention it, I can actually think of no reason why it's in there. I have the feeling that it's purely for cargo culting reasons. I remember times when we always added bundler as a dev dep for every gem. It might have had something to do with "being able" to invoke the devtools rake tasks (that e.g. use bundler groups). Anyways, I'm not at all sure wether that would be a valid reason in the first place either. Surely nowadays it's safe to assume that bundler is kinda omnipresent in the ruby dev world and I agree that the explicit dependency could therefore be dropped @mbj what are your thoughts on this?